src/staff_augmentation/components/GetFreeQuote/styles.module.scss

```diff
@@ -5,25 +5,50 @@
   display: flex;
   flex-direction: column;
   padding: 50px;
-  width: 1207px;
+  width: 100%;
   border-radius: 0px 0px 52px 52px;
   margin: auto;
   background: $secondaryColor;
+
+  @include for-tablet-up {
+    width: 648px;
```

Comment:
Use `max-width` for a fully responsive implementation. Combined with `width: 100%` from line 8, it'll achieve the same result you're looking for, but if there's not enough space, it'll be narrower.